dist/manifest.json

```diff
@@ -18,13 +18,6 @@
     "webRequestBlocking",
     "http://*/*",
     "https://*/*"
-  ],
-
-  "content_scripts": [
-    {
-      "matches": ["<all_urls>"],
-      "js": ["common.js", "content_script.js"]
-    }
```

Comment:
this is needed only if we are using content script functionality, which at this point we are not. So I removed it for some clarity. will add it back once we decide how to show the recorded log. I'm thinking a new tab is better. We can discuss this with some prototypes.